### PR TITLE
Added contribution guide

### LINKED ISSUE #14: Contribution guide
Create a `CONTRIBUTING.md` file with instructions on how to clone and setup the repo locally.
The guide should also be linked from `README.md` under *Contribution* section

### PR DESCRIPTION
**Pull request by Manas-Kenge**
Fixes #14

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+#### Solve an issue
+
+Scan through our [existing issues](https://github.com/khattakdev/capture/issues) to find one that interests you. You can narrow down the search using `labels` as filters. If you find an issue to work on, you are welcome to open a PR with a fix.
+
+### Make Changes
+
+#### Make changes in a codespace
+
+Make your code changes on this branch. Ensure your changes adhere to our coding guidelines.
```
Comment (khattakdev) [inline CONTRIBUTING.md:36]: Add more info on how can one use codespace to make changes.

Comment (Manas-Kenge) [reply CONTRIBUTING.md]: I have made the required changes, please check them.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+2. Run Commands: Execute commands in the integrated terminal to build, test, or run your project.
+
+3. Version Control: Use Git commands to manage your changes. You can commit, create branches, and push your commits directly to your GitHub repository. Codespaces also integrates with GitHub Pull Requests, allowing you to create and review pull requests from within the environment.
+
+4. Real-Time Collaboration: If you're working on a collaborative project, multiple users can access and work on the same Codespaces environment simultaneously, making real-time collaboration easy.
+
+#### Make changes locally
+
+1. Fork the repository.
+- Using GitHub Desktop:
```
Comment (khattakdev) [inline CONTRIBUTING.md:59]:
```suggestion
- **Using GitHub Desktop:**
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,91 @@
...
+  - [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/getting-started-with-github-desktop) will guide you through setting up Desktop.
+  - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
+
+- Using the command line:
```
Comment (khattakdev) [inline CONTRIBUTING.md:63]:
```suggestion
- **Using the command line:**
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,91 @@
...
+  - Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
```
Comment (khattakdev) [inline CONTRIBUTING.md:64]:
```suggestion
  - To set up a working development environment, just fork the project git repository.
  - Open your terminal and navigate to the directory where you want to store the project. Then, run the following command to clone the repository: git clone https://github.com/yourusername/capture.git
- Navigate into the project directory: `cd capture`
- Use npm to install the project's dependencies: `npm install`
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,91 @@
...
+
+2. Install or update to **Node.js**, at the version specified in `.node-version`. For more information, see the development guide.
+
+3. Create a working branch and start with your changes!
```
Comment (khattakdev) [inline CONTRIBUTING.md:68]:
```suggestion

```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,91 @@
...
+
+1. Fork the repository.
+- Using GitHub Desktop:
+  - [Getting started with GitHub Desktop](https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/getting-started-with-github-desktop) will guide you through setting up Desktop.
+  - Once Desktop is set up, you can use it to [fork the repo](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/cloning-and-forking-repositories-from-github-desktop)!
+
+- Using the command line:
+  - Fork the repo so that you can make your changes without affecting the original project until you're ready to merge them.
+
+2. Install or update to **Node.js**, at the version specified in `.node-version`. For more information, see the development guide.
```
Comment (khattakdev) [inline CONTRIBUTING.md:66]:
```suggestion
```

**Review by khattakdev — CHANGES_REQUESTED** [review]

**Pull request merged**